specs/dynamicdependencies/DynamicDependencies.md

```diff
@@ -1250,14 +1252,17 @@ STDAPI MddGetResolvedPackageFullNameForPackageDependency(
 STDAPI MddGetIdForPackageDependencyContext(
     _In_ MDD_PACKAGEDEPENDENCY_CONTEXT packageDependencyContext,
     _Outptr_result_maybenull_ PWSTR* packageDependencyId);
+
+/// Return the package graph's current generation id.
+STDAPI_(UINT32) MddGetGenerationId() noexcept;
```

Comment:
Is the `noexcept` necessary here? Leaking exceptions across a DLL boundary is utterly verboten, so if this is an export I'm not sure it's required.  (Some compilers, like cl, have an "assume extern-c methods, like this one, don't throw.")

Reply:
`noexcept` is actually on all the existing exports, added during a code review due to feedback last year by you, Jevan or EHO (I forgot who). It's here for consistency and is harmless (at least, if specified). I haven't heard if VS assumes extern-C implies `noexcept`. I'll look into it for a future PR, easy enough to remove if redundant.